flink-examples/flink-java-examples/pom.xml

```diff
@@ -319,7 +319,32 @@ under the License.
 							</includes>
 						</configuration>
 					</execution>
-					
+
```

Comment:
This will produce another jar file in the `examples/` directory, right?

If so, I'm against this because its very hard for beginners to understand the purpose of this example.